bundles/org.openhab.binding.sonnen/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="sonnen"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sonnen Thing Type -->
+	<thing-type id="battery">
+		<label>sonnen Binding Thing</label>
+		<description>Thing for sonnen battery connection</description>
+
+		<channels>
+			<channel id="channelBatteryCharging" typeId="isBatteryCharging"/>
+			<channel id="channelBatteryDischarging" typeId="isBatteryDischarging"/>
+			<channel id="channelConsumption" typeId="consumption"/>
+			<channel id="channelGridFeedIn" typeId="gridFeedIn"/>
+			<channel id="channelSolarProduction" typeId="solarProduction"/>
+			<channel id="channelBatteryLevel" typeId="batteryLevel"/>
+			<channel id="channelFlowConsumptionBattery" typeId="isFlowConsumptionBattery"/>
+			<channel id="channelFlowConsumptionGrid" typeId="isFlowConsumptionGrid"/>
+			<channel id="channelFlowConsumptionProduction" typeId="isFlowConsumptionProduction"/>
+			<channel id="channelFlowGridBattery" typeId="isFlowGridBattery"/>
+			<channel id="channelFlowProductionBattery" typeId="isFlowProductionBattery"/>
+			<channel id="channelFlowProductionGrid" typeId="isFlowProductionGrid"/>
```

Comment:
Please remove "channel" prefix from all your channel ids.

Reply:
Are you sure? I did it exactly like in my other binding which is already merged in openhab-addons. See [here](https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.haassohnpelletstove/src/main/resources/OH-INF/thing/thing-types.xml)

Reply:
Yes, look at all other bindings. I guess you will not find this orefix for 99% of the channels.

Reply:
ok